docs/frequency-analysis.md

```diff
@@ -0,0 +1,11 @@
+Frequency analysis, which is also known as counting letters, is a useful study within the field of computer science. This is due to its function because it is being capable of outputting the frequency of letters or groups of letters in a ciphertext.
```

Comment:
Similar here, please make sure to avoid long lines and break up the markdown text